### PR TITLE
Add Groupon, HockeyApp, IconJar, Mix, Safari

### PR DESCRIPTION
**Pull request by abenjamin765**
- Add Groupon
- Add HockeyApp
- Add IconJar
- ~~Add kaiOS~~
- Add Mix
- Add Safari
- Remove Stumbleupon

### CODE REVIEW

**Comment by ericcornelissen** [comment]
@abenjamin765, you still did not really remove kaiOS, as the SVG is still part of your PR. And to address your comment in #943, I think in order for Travis to like the kaiOS addition, the name of the icon should be in all lowercase :wink:

However, you might have some trouble with that given how some OSs and git handle lettercase in filename, [this](https://stackoverflow.com/questions/10523849/changing-capitalization-of-filenames-in-git) may be helpfull

**Comment by abenjamin765** [comment]
I removed the file. I had everything lowercase in the json file and the filename in fd481a4. No luck. I'll open a separate PR when I get it working.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -355,6 +355,11 @@
             "hex": "2A73CC",
             "source": "https://about.coursera.org/press"
         },
+        {
+            "title": "Creative Cloud",
```
Comment (birjolaxew) [inline _data/simple-icons.json]: This should be "Adobe Creative Cloud", similar to how Photoshop is "Adobe Photoshop". Remember to update the SVG too :)

File: _data/simple-icons.json [hunk 2]
```diff
@@ -355,6 +355,11 @@
...
+            "hex": "DA1F26",
+            "source": "https://www.adobe.com/legal/permissions/icons-web-logos.html"
```
Comment (birjolaxew) [inline _data/simple-icons.json]: Note that the [Creative Cloud-specific guidelines](https://www.adobe.io/apis/creativecloud/creativesdk/docs/websdk/adobe-creative-sdk-for-web_master/branding-guidelines.html) contains an "on-tile" and an "off-tile" variant (with the off-tile version being the inverse of the version in this PR). However the [official website](https://www.adobe.com/dk/creativecloud.html) uses the variant in this PR, and I therefore vote that we stick with that.

File: _data/simple-icons.json [hunk 3]
```diff
@@ -355,6 +355,11 @@
             "hex": "2A73CC",
             "source": "https://about.coursera.org/press"
         },
+        {
+            "title": "Creative Cloud",
+            "hex": "DA1F26",
```
Comment (birjolaxew) [inline _data/simple-icons.json]: The official color seems to be `#D41818`:

```css
.adobe-icon-creativecloud {
    color: #d41818
}
```

File: _data/simple-icons.json [hunk 4]
```diff
@@ -745,6 +750,11 @@
             "hex": "1E8CBE",
             "source": "https://automattic.com/press"
         },
+        {
+            "title": "Groupon",
+            "hex": "82B742",
```
Comment (birjolaxew) [inline _data/simple-icons.json]: The official color - both from their brand playbook and from their website - is `#53A318`

File: _data/simple-icons.json [hunk 5]
```diff
@@ -800,6 +810,11 @@
             "hex": "0052CC",
             "source": "https://www.atlassian.com/company/news/press-kit"
         },
+        {
+            "title": "HockeyApp",
+            "hex": "1C9DD8",
```
Comment (birjolaxew) [inline _data/simple-icons.json]: The color [is stated](https://hockeyapp.net/brand-guidelines/) to be `#009EE1`

File: _data/simple-icons.json [hunk 6]
```diff
@@ -830,6 +845,11 @@
             "hex": "CC2929",
             "source": "https://support.humblebundle.com/hc/en-us/articles/202742060-Bundle-Logos"
         },
+        {
+            "title": "IconJar",
+            "hex": "109CE8",
```
Comment (birjolaxew) [inline _data/simple-icons.json]: The color is defined [in their press kit](https://geticonjar.com/press-kit/) to be `#16A5F3`

**Comment by birjolaxew** [comment]
:+1: Thanks for all your work on this! It is much appreciated.

*(note that the last commit also removes StumbleUpon as discussed in #931)*

**Pull request merged**